lib/reporters/verbose.js

```diff
@@ -3,6 +3,7 @@ var prettyMs = require('pretty-ms');
 var figures = require('figures');
 var chalk = require('chalk');
 var plur = require('plur');
+var repeatString = require('repeat-string');
```

Comment:
Ha! I knew you'd written one! It didn't show up when searching for [string repeat](https://www.npmjs.com/search?q=string+repeat) :smile: